lambda_src/lambda_function.py

```diff
@@ -139,7 +141,7 @@ def sync_flow(event: Any, context: Any = None) -> Dict[Text, Any]:
         res_data.append(
             [
                 row_number,
-                zip(dumps(row_result)) if response_encoding == 'gzip' else row_result,
```

Comment:
once this is removed, `response_encoding` becomes an unused variable. which editor do you use? ideally, it should pick up vestigial code like this.

Reply:
I am using VS CODE Editor. Changed the code, deleted the unused variable

Reply:
does VS Code show you anything as off? e.g. also with the "zip" import, that should now be unused?

Reply:
It didn't shown any underline so not removed that, now modified it and removed zip import

Reply:
any time you work with a new language you should spend some time figuring out the best tooling around it. in this case, it seems VSCode doesn't have it set up by default but you should follow [these steps](https://code.visualstudio.com/docs/python/linting) to install `pylint` so that it helps you with things like this: https://pylint.pycqa.org/en/latest/user_guide/messages/warning/unused-import.html

Reply:
Sure Andrey I will remember this in Future